keywords/brain-interface.md

```diff
@@ -1,7 +1,17 @@
 ## Brain Interfaces
 
-![Brain Interfaces](images/brain-interface-on_line.png)[^Angelsimon]
+![Brain Interfaces](../images/brain-interface-on_line.png)[^Angelsimon]
```

Comment:
@mslineon check the instructions, to be properly displayed on Github, images need to be linked as

```
![Brain Interfaces](images/brain-interface-on_line.png)
```